packages/esm-outpatient-app/src/active-visits/active-visits-table.test.tsx

```diff
@@ -18,6 +18,14 @@ jest.mock('./active-visits-table.resource.ts', () => {
   };
 });
 
+jest.mock('@openmrs/esm-framework', () => {
```

Comment:
`openmrsFetch` is provided by the default `@openmrs/esm-framework` mock, so this isn't necessary for the test to pass.